### DISCUSSION TITLE
Can [ObservableProperty] generate code from base class that already implementation the INotifyPropertyChanged interface？

### DISCUSSION THREAD

**Discussion by ZSYMAX** [CommunityToolkit/dotnet#867] [General]
![class](https://github.com/CommunityToolkit/dotnet/assets/38749563/fbc51a8b-3bb0-40a3-ad5d-380fbf926f58)

**Comment by hansmbakker** [comment]
Duplicate of https://github.com/CommunityToolkit/dotnet/discussions/336
